lib/config/sass-lint.yml

```diff
@@ -18,23 +18,23 @@ rules:
 
   # Disallows
   no-debug: 1
-  no-duplicate-property: 1
+  no-duplicate-properties: 1
   no-empty-rulesets: 1
   no-extends: 0
   no-ids: 1
   no-important: 1
   no-warn: 1
+  no-color-keywords: 1
+  no-invalid-hex: 1
+  no-css-comments: 1
```

Comment:
As these are our default values and we have a separate [sample config](https://github.com/sasstools/sass-lint/blob/master/docs/sass-lint.yml), I don't really care if it's alphabetical or not.